_posts/2022-05-12-fluentcon-announcement.md

```diff
@@ -0,0 +1,30 @@
+---
+layout: post
+title:  "FluentCon EU Announcement"
+authors:
+- anurag_gup
+date: 2022-05-12
```

Comment:
Looks good. Only thing - since you have tomorrow's date in there, it may not show up until then - do you want to edit? @brijos